Makefile

```diff
@@ -66,7 +66,33 @@ proto-crux-ui:
 .PHONY: all
 all: | protogen docs
 
-
 .PHONY: build-proto-image
 build-proto-image:
 	docker build -t ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.16 images/proto
+
+.PHONY: release
+release:
+	@echo "Version will be: $(version)"
```

Comment:
Tested locally, not printed the version :D

Reply:
Seems like a Mac issue, worked fine for me. :smiling_imp:

Reply:
removed the @